### PR TITLE
Navbar

### PR DESCRIPTION
**Pull request by AdamTahiri**

### CODE REVIEW

File: src/Components/Navbar.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import React from "react";
+import { NavLink } from "react-router-dom";
+import Homepage from "./Homepage";
+
+const Navbar = () => {
+  return (
+    <nav>
+      <NavLink exact to="/"></NavLink>
```
Comment (AdamTahiri) [inline src/Components/Navbar.js]: Have youtube be the home page.

File: src/App.js [hunk 2]
```diff
@@ -3,6 +3,7 @@ import './App.css';
 function App() {
   return (
     <div className="App">
+      <Navbar/>
```
Comment (AdamTahiri) [inline src/App.js]: Implement Route and Switch to each component

Comment (ObaidRustemi) [reply src/App.js:6]: Implement Route and Switch to each component

**Review by ObaidRustemi — CHANGES_REQUESTED** [review]
change stuff

**Pull request merged**